airgun/entities/computeresource.py

```diff
@@ -91,9 +91,13 @@ def vm_poweroff(self, entity_name, vm_name):
         if vm['Power'].widget.read() == 'On':
             vm['Actions'].widget.click(handle_alert=True)
 
-    def vm_import(self, entity_name, vm_name, hostgroup, location):
+    def vm_import(self, entity_name, vm_name, hostgroup, location, org=None, name=None):
         """Imports the specified VM"""
         view = self.navigate_to(self, 'VMImport', entity_name=entity_name, vm_name=vm_name)
+        if name:
+            view.fill({'host.name': name})
+        if org:
+            view.fill({'host.organization': org})
```

Comment:
Maybe I was not able to comment on the original PR, I feel there is not need to pass the extra parameters, you can pass everything from the test itself. 

e.g. 

`python 
values = {'host.name': name, 'host.hostgroup': hostgroup, 'host.location': location}}
session.ComputeResource().vm_import(entity_name, values)
`